osu.Game/Skinning/LegacyAccuracyCounter.cs

```diff
@@ -24,16 +27,19 @@ public LegacyAccuracyCounter(ISkin skin)
             Margin = new MarginPadding(10);
 
             this.skin = skin;
+            scorePrefix = skin.GetConfig<LegacySkinConfiguration.LegacySetting, string>(LegacySkinConfiguration.LegacySetting.ScorePrefix)?.Value ?? "score";
```

Comment:
This is probably going to be used in more places than just here, so I guess we should revert to doing things the way I had them:

rather than `new LegacySpriteText` below, it should be `skin.GetDrawable(new HUDComponent(ScoreText)`.

then the config lookups etc. would be inside that lookup. you can see the logic in the left hand of the diff here: https://github.com/ppy/osu/pull/10504/commits/37e9f331ad78fb9ff10701888169e5ea47ddea7a

Reply:
I'm not sure I understand. Are you suggesting to move the `GetConfig` calls into `LegacySkin.GetDrawableComponent`, which is then called by the ``CreateSpriteText()`` method? If so, would `GetDrawableComponent` have to create and return that `LegacySpriteText` (which I'd also have to cast to a `OsuSpriteText`)?
I'm kinda confused right now.

Reply:
The linked commit should explain quite well (it's a full implementation of what I'm asking for here).

Reply:
Okay, I've implemented what I understood from your comment. Please tell me if this isn't what you asked for.